specification/cognitiveservices/data-plane/Speech/VideoTranslation/preview/2024-05-20-preview/VideoTranslation.json

```diff
@@ -4,13 +4,6 @@
     "title": "Video Translation",
     "version": "2024-05-20-preview",
     "description": "Video translation API to translate video from source locale to target locale.",
-    "contact": {
```

Comment:
@markcowl Do you know if removal of this info is a breaking change? The `@info` decorator is deprecated, and looking through a sampling of specs I see that this info is not present when using TypeSpec to generate the swagger specs.

I think this spec was just added yesterday (https://github.com/Azure/azure-rest-api-specs/pull/29572)

Reply:
for `@info` I just kept it and suppressed saying they wanted those fields for now.